proto/osmosis/gamm/v1beta1/query.proto

```diff
@@ -165,7 +164,6 @@ message QuerySwapExactAmountInResponse {
 
 //=============================== EstimateSwapExactAmountOut
 message QuerySwapExactAmountOutRequest {
-  string sender = 1 [ (gogoproto.moretags) = "yaml:\"sender\"" ];
   uint64 pool_id = 2 [ (gogoproto.moretags) = "yaml:\"pool_id\"" ];
```

Comment:
This change is going to be a breaking change anyhow, Can we update the field numbers? (e.g pool_id -> 1)

Reply:
Ah just checked the convo above resolving this